src/modifiercontext.ts

```diff
@@ -7,6 +7,7 @@
 
 import { Vex } from './vex';
 import { StaveNote } from './stavenote';
+import { GlyphNote } from './glyphnote';
```

Comment:
This line can be deleted. `GlyphNote` is not used.

Reply:
done

Reply:
The line is still there also